src/state/finishedGames/finishedGameReducer.ts

```diff
@@ -11,29 +11,30 @@ const finishedGameReducer = ( state = initialState, action: GenericAction ) => {
     switch (action.type) {
 
         case SAVE_GAME_DATA: {
-            let clone = cloneState( state );
-
+            const clone = cloneState( state );
```

Comment:
We should start to use immer to avoid this state.clone stuff

Reply:
What do you mean with immer?

Reply:
https://github.com/mweststrate/immer

Reply:
Ok. I will try it out!

Reply:
I switched the code to immer.